file/1.5/modules/ROOT/pages/index.adoc

```diff
@@ -1,62 +1,34 @@
 = File Connector - Mule 4
 :page-aliases: connectors::file/file-connector.adoc
 
+Anypoint Connector for File (File Connector) manages files and folders on a locally mounted file system. The connector's main features include:
 
+* The ability to read files or fully list directory contents on demand.
+* Support for common File operations such as creating directories, reading, writing, copying, moving, renaming, deleting and listing files.
+* Support for locking files.
+* File matching functionality.
+* A design that is consistent with FTP Connector and SFTP Connector.
 
-File Connector v1.5
+== Representation of File Attributes
 
-Anypoint Connector for File (File Connector) handles files and folders on a locally mounted file system. Its main features include:
+Mule 4 provides a message structure of a payload and attributes in which the data moves around, making that payload a file content. The attributes provide an object that contains metadata on the payload and information such as the file’s name, path, size, and timestamps. For FTP Connector and SFTP Connector, these attributes are known as  _file attributes_.
 
-// INCLUDE FOR file, ftp, and sftp connectors
-include::partial$common-file-actions.adoc[]
+The Mule message contains the concepts of encoding and MIME type. These describe the format in which a payload is represented. The file attributes describe a file’s encoding and MIME type.
```

Comment:
I don't think "contains the concepts of encoding and MIME type" is quite right. Maybe "The Mule message specifies the encoding and MIME types, which describe the format in which the payload is represented."

Then you could say, "The file attributes specify a file's encoding and MIME type."